src/meta/Dimensions.ts

```diff
@@ -0,0 +1,63 @@
+import { Base } from './Base';
+
+export interface DimensionResults {
```

Comment:
Yeah, like I said, I can see where you're coming from. I can go ahead and split these into groups. Is `marginTop` an actual property or is only on `style.marginTop`? If we start including style properties, maybe we should come up with a list of what we'll need to include, i.e., margins, padding, borders, etc. I think all of those affect position and size?